### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by maulanarasoky**

### CODE REVIEW

File: Startup's To Do/src/com/company/Main.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+    static Scanner input = new Scanner(System.in);
+    static ArrayList<Pair> array = new ArrayList<Pair>();
+    static int bayar = 0;
+
+    public static void main(String[] args) {
+        int pilih;
+        do {
+            System.out.println("===== Startup's To Do =====");
+            System.out.println("Menu\n1. Masukkan Project Baru\n2. Hapus Project\n3. Tampilkan Daftar Project\n4. Jumlah bayaran\n5. Exit");
```
Comment (blinfoldking) [inline Startup's To Do/src/com/company/Main.java:16]: this line is too long, you can split it with multiple statement

File: Startup's To Do/src/com/company/Main.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+                    System.out.print("Masukkan nama proyek : ");
+                    String namaProyek = input.next();
+                    System.out.print("Masukkan jumlah bayaran : ");
+                    int bayaran = input.nextInt();
+
+                    Pegawai pegawai = new Pegawai(nama, umur);
+                    Klien klien = new Klien(namaKlien, namaPerusahaan);
+                    Proyek proyek = new Proyek(namaProyek, bayaran);
+                    Pair pair = new Pair(pegawai, klien);
+                    Pair pair2 = new Pair(pair, proyek);
```
Comment (blinfoldking) [inline Startup's To Do/src/com/company/Main.java:38]: avoid undescriptive variable like this in the future

File: Startup's To Do/src/com/company/Main.java [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+                    array.add(pair2);
+                    bayar = bayar + bayaran;
+                    break;
+                case 2:
+                    Proyek prk = (Proyek) array.get(array.size() - 1).getKanan();
```
Comment (blinfoldking) [inline Startup's To Do/src/com/company/Main.java:43]: same here

File: Startup's To Do/src/com/company/Proyek.java [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    String namaProyek;
+    int bayaran;
+
+    public Proyek(String namaProyek, int bayaran) {
+        this.namaProyek = namaProyek;
+        this.bayaran = bayaran;
+    }
+
+    public String getNamaProyek() {
+
```
Comment (blinfoldking) [inline Startup's To Do/src/com/company/Proyek.java:13]: trailing whitespace

File: Startup's To Do/src/com/company/Klien.java [hunk 5]
```diff
@@ -0,0 +1,32 @@
+package com.company;
+
+public class Klien {
+    String namaKlien, perusahaan;
+
+    public String getNamaKlien() {
+        return namaKlien;
+    }
+
+    public Klien(String namaKlien, String perusahaan) {
```
Comment (blinfoldking) [inline Startup's To Do/src/com/company/Klien.java:10]: always put constructor on top before any other method

File: Startup's To Do/src/com/company/Main.java [hunk 6]
```diff
@@ -0,0 +1,61 @@
+package com.company;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Main {
+
+    static Scanner input = new Scanner(System.in);
```
Comment (blinfoldking) [inline Startup's To Do/src/com/company/Main.java:8]: avoid static declaration if it not used in any other instance

File: Startup's To Do/src/com/company/Pegawai.java [hunk 7]
```diff
@@ -0,0 +1,34 @@
+package com.company;
+
+public class Pegawai {
```
Comment (blinfoldking) [inline Startup's To Do/src/com/company/Pegawai.java:3]: i think you should use struct instead

Comment (blinfoldking) [reply Startup's To Do/src/com/company/Pegawai.java:3]: struct is more suitable for simple abstraction

File: Startup's To Do/src/com/company/Pegawai.java [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+    String nama;
+    int umur;
+
+    public Pegawai(String nama, int umur) {
+        this.nama = nama;
+        this.umur = umur;
+    }
+
+    public String getNama() {
+
```
Comment (jeremyfl) [inline Startup's To Do/src/com/company/Pegawai.java:13]: Whitespace, should automaticly deleted with Java formatter that I used

Comment (blinfoldking) [reply Startup's To Do/src/com/company/Pegawai.java:13]: could you please elaborate the name of formatter that you use

Comment (jeremyfl) [reply Startup's To Do/src/com/company/Pegawai.java:13]: It's, Prettier - Code formatter in VSCode.

**Pull request merged**